### PR TITLE
Add CLI joke

### PR DESCRIPTION
**Pull request by linda-lai**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -713,3 +713,9 @@ What's the best tool for automatically ignoring long email threads about tech bu
 **A:** They always promise to callback
 
 ---
+
+**Q:** Why is the command line trustworthy?
```
Comment (styfle) [inline readme.md]:
```suggestion
**Q:** Why do developers mixup Terminals and Polygraphs?
```

File: readme.md [hunk 2]
```diff
@@ -713,3 +713,9 @@ What's the best tool for automatically ignoring long email threads about tech bu
...
+
+**A:** Because they can see a lie (CLI)
```
Comment (styfle) [inline readme.md]:
```suggestion
**A:** Because they both can see a lie (CLI)
```

**Comment by styfle** [comment]
I like it! But I left some suggestions that I think will improve the joke 😉

File: readme.md (hunk 2 above)
Comment (linda-lai) [reply readme.md]: Sure! I like the changes too 😅

**Pull request merged**